corehq/apps/export/static/export/js/export_list.js

```diff
@@ -1,8 +1,22 @@
+/**
+ *  This module contains the knockout models that control the "list" part of the exports list.
+ *  This serves all export types: form vs case and exports vs daily saved vs dashboard feeds.
+ *  It does NOT contain the logic for adding a new export, which is also done on the export list page.
+ *
+ *  There are three models in this file:
+ *      exportModel represents an individual export: name, description, case type, etc.
+ *      exportPanelModel represents a set of exports. Each of these is displayed in the UI as
+ *          a panel with independent pagination.
```

Comment:
nit: "page" is a more familiar term to me than "panel"

Reply:
I don't understand - how would you phrase this?

Reply:
`exportPageModel` or something.  When I read `exportPanelModel`, I thought it was a panel representing an individual export, rather than a set of exports.